### PR TITLE
Train RGB

### PR DESCRIPTION
**Pull request by markus-hinsche**
Related to https://github.com/Welthungerhilfe/cgm-ml/pull/413

**Changes:**
- add pretrained VGG16 arch (with imagenet weights)
- add fine-tuning code
- add data augmentation

**Results:**
On this dataset, taking only depthmaps yields a val_mae of `1.895cm` or smaller.
On this dataset, using RGB yields a a val_mae of `4.54cm` or larger. This is clearly worse

These loss curves ([Experiment](https://ml.azure.com/experiments/id/3c83013d-c290-4eaf-8264-de9f448b210d?wsid=/subscriptions/9b5bbfae-d5d1-4aae-a2ca-75159c0c887d/resourceGroups/cgm-ml-prod-we-rg/providers/Microsoft.MachineLearningServices/workspaces/cgm-ml-prod-we-azml&tid=3a27c573-ec1a-4734-9cd3-3208af51794b) Run 46-49) show the brown baseline (depthmap-only). The other curves are the RGB runs with different configurations (bgr, fine-tune, data augmentation; each on or off)
image

**Details about data augmentation**
original:
image

data augmentation:
image

### CODE REVIEW

**Comment by markus-hinsche** [comment]
Closing because of unfavorable results of this RGB experiment compared to depthmap-only baseline.
